tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "jsx": "react",
     "lib": [
       "es6",
+      "es7",
```

Comment:
I added because It errors without es7 support when using Array.includes() with string[] type. I think Array.includes() is not that super needed, I changed all of it to just use Array.filter() now.

Reply:
I've looked through how to write rule's metadata(exportNameRule, functionNameRule)... didn't get how to write it.  I wrote optionsExample which I thought I could write.